### ISSUE TITLE
config reference with Windows Version 2.4

### ISSUE THREAD

**Issue by SBFspot** [SBFspot/SBFspot#15]
Hi,
I cannot get the tool running, because it does not find the config file.
D:\Downloads\SMAspot>SMAspot.exe
SMAspot V2.4.0
Yet another tool to read power production of SMA solar inverters
(c) 2012-2014, SBF (https://smaspot.codeplex.com)
Compiled for Windows 32 bit

Commandline Args:
Error! Could not open file SMAspot.exeSMAspot.cfg

Trying to pass the cfg parameter:
D:\Downloads\RaspberryPi\SMAspot>SMAspot.exe -cfgSMASpot.cfg
SMAspot V2.4.0
Yet another tool to read power production of SMA solar inverters
(c) 2012-2014, SBF (https://smaspot.codeplex.com)
Compiled for Windows 32 bit

Commandline Args: -cfgSMASpot.cfg
Error! Could not open file SMAspot.exeSMASpot.cfg

btw: I love SMASpot :-)

#### Attachments
[wsi7g051.4211F-tst-CPl008.txt](https://www.codeplex.com/Download/AttachmentDownload.ashx?ProjectName=SBFspot&WorkItemId=8&FileAttachmentId=792785)
[sunny.cmd](https://www.codeplex.com/Download/AttachmentDownload.ashx?ProjectName=SBFspot&WorkItemId=8&FileAttachmentId=792830)

#### This work item was migrated from CodePlex
CodePlex work item ID: '8'
Assigned to: 'SBF'
Vote count: '1'

**Comment by SBFspot** [comment]
[SBF@11/02/2014]
I'm afraid I introduced a new bug :-(
You can work around by giving the full path to the cfg file:
> D:\Downloads\RaspberryPi\SMAspot>SMAspot.exe -cfgD:\Downloads\RaspberryPi\SMASpot.cfg

Don't forget to add you timezone (e.g. timezone=Europe/Brussels)
See date_time_zonespec.csv

**Comment by SBFspot** [comment]
[ReneHezser@11/02/2014]
I renamed the config file. That way it worked for me :-)

Now the tool complains about the missing date_time_zonespec.csv, even if the file is in the same folder
```
D:\Downloads\RaspberryPi\SMAspot>SMAspot.exe -cfgSMAspot.cfg
SMAspot V2.4.0
Yet another tool to read power production of SMA solar inverters
(c) 2012-2014, SBF (https://smaspot.codeplex.com)
Compiled for Windows 32 bit

Commandline Args: -cfgSMAspot.cfg
Unable to open 'date_time_zonespec.csv'
```
The directory
```
 Directory of D:\Downloads\RaspberryPi\SMAspot

11.02.2014  10:38    <DIR>          .
11.02.2014  10:38    <DIR>          ..
22.09.2012  16:04            35.855 date_time_zonespec.csv
12.02.2008  20:15           270.336 libcurl.dll
18.03.2010  08:15           770.384 msvcr100.dll
02.02.2014  20:11           301.568 SMAspot.exe
11.02.2014  10:37             5.304 SMAspot.exeSMAspot.cfg
11.02.2014  10:06    <DIR>          smaspot_48d42c3c76a1
20.07.2005  18:05            75.264 zlib1.dll
               6 File(s)      1.458.711 bytes
               3 Dir(s)  369.473.818.624 bytes free
```

**Comment by SBFspot** [comment]
[SillieWimons@11/02/2014]
hej rene,   don't know whether this has influence (guess not)
   i did not have these issues
   but i do execute SMAspot from a directory on the C: drive
         and i execute SMAspot from a .bat (read script)
   i will repeat exactly your set-up to validate the results :-)

note that if you solve the timezone issue then you will get an error on the tagfiles
   you must also copy the 6 files "TagList*.txt" (from sources) to the prog.dir (the one with SMAspot.exe)

to sbf, i did not have the above errors, 
   but i can confirm that there is a difference in reaction when more then one option is given in CLI

kr wim

**Comment by SBFspot** [comment]
[SillieWimons@11/02/2014]
hej, rene
used your setup - starting SMAspot from a command-box
had the same problems as you encountered

the workaround is that you must make a copie of each file and preceed them by SMAspot.exe
files: SMAspot.cfg; date_time_zonespec.csv; TagListEN-US.txt

after that (i've kept the original files as well) SMAspot executes normal (sic)

txt file will follow to show the results of my tests

kr wim

**Comment by SBFspot** [comment]
[SillieWimons@11/02/2014]
dear rene and luc, sending a file directlty as attachment did give problems to codeplex

**Comment by SBFspot** [comment]
[ortmueller@11/02/2014]
Hello,

use the absolute path to run smaspot.exe:
C:\SMA-spot.v240-vist\SSpot-ma.240b.prg\SMAspot.exe

this solved all Probelms for me.

At this time no csv is written, i create a small batch, be free to use at own risk, and change the smaspot dir.

**Comment by SBFspot** [comment]
[BenniGotchi@14/02/2014]
If you want to start SMAspot from command line, it's also possible, to start it with
.\SMAspot.exe
if you are in the SMAspot directory.

important thing is, that there is at least one "\" in the call.

**Comment by SBFspot** [comment]
[ReneHezser@16/02/2014]
Fixed with 2.4.1 :-)

**Comment by SBFspot** [comment]
**Issue closed by _SBF_ with comment**
Thanks for the feedback
Closing issue.

**Reason closed**
Fixed
